splunk_eventgen/__main__.py

```diff
@@ -6,71 +6,128 @@
 
 import argparse
 import errno
+import logging
 import os
 import shutil
 import sys
 import time
 
 import requests
 
+import __init__ as splunk_eventgen_init
+import eventgen_core
+
 FILE_LOCATION = os.path.dirname(os.path.abspath(__file__))
 path_prepend = os.path.join(FILE_LOCATION, 'lib')
 sys.path.append(path_prepend)
-import __init__ as splunk_eventgen_init
```

Comment:
I have not tested whether this will cause import error since the path is inserted into sys path before import.

Reply:
Good catch, I'll mark these lines so they don't get reformatted.